### DISCUSSION TITLE
Community Discord Server

### DISCUSSION THREAD

**Discussion by nlogozzo** [AvaloniaUI/Avalonia#5356] [General]
Hi all,

I've decided to start a community discord server for Avalonia users and developers alike. The aim of the sever is to be an easy way for developer to get together and discuss issues, get/give help, share projects, collaborate together, and just be a fun community for developers. I look forward to see you all there and growing the server together :)

Join Link:
https://discord.gg/gzsBrpNzh7

**Comment by FoggyFinder** [comment]
Those who use Tor browser (or even just use proxy) are unable to use Discord.

**Reply by nlogozzo** [reply]
Sorry to here that. But discord is a platform used by many developers and is a great place for beginners to use an already familiar platform to get started and experienced developers to use as well to assist beginners and collaborate together. Again this is just a community made server for those that are interested in joining :)

**Reply by Splitwirez** [reply]
Discord offers a desktop client as well. Granted, it *is* Electron-based, but it's separate from the user's browser of choice nonetheless...that renders the user's preferred browser/use of s proxy a non-issue, does it not?

**Reply by FoggyFinder** [reply]
As far as I can tell `Electron-based` means awful. But I'll try to give it a chance

**Reply by Splitwirez** [reply]
Some people would use `Electron-based` as a roast, yes...but not me. Many Electron apps are notorious for their abysmal performance, but ever since using Visual Studio Code, I've...hesitated to blame Electron itself for that. Discord's client is...mediocre in this regard.

I only mentioned the fact that Discord's client is Electron-based because I wasn't sure if the fact that it's still arguably a web browser-ish thing might be a problem or something. I'm not at all familiar with proxies, personally. (Is "proxies" even the correct plural form?)

**Reply by FoggyFinder** [reply]
It seems like I was wrong. The issue isn't related to proxy. I've tried to use desktop app but unfortunately the result was the same as for web-version: it didn't allow me to proceed. The additional phone confirmation is required. Obviously, I don't want to share my phone number without a good reason. Probably my mail looks suspicious, dunno. I'll try to create another e-mail account and give it another shot.

**Reply by Splitwirez** [reply]
That's...really weird. It never forced me to share my phone number...hopefully that new email address will do the trick.

**Comment by kekekeks** [comment]
IIRC there is already a channel on .NET Foundation discord

**Reply by nlogozzo** [reply]
I don't think there is a .NET Foundation discord...never heard of one, but I could be wrong. Either way, this server is a group of dedicated Avalonia developers that could address specific issues and problems, instead of other discords with generic C# developers.

**Reply by FoggyFinder** [reply]
Here:

https://aka.ms/dotnet-discord

(the link above is taken from ReadMe )

**Reply by Splitwirez** [reply]
Well...is it a *problem* if there's another Discord server, this one specifically dedicated to Avalonia?

**Reply by FoggyFinder** [reply]
Nope. But maybe it would be better to have one place rather than many small ones?

**Reply by Splitwirez** [reply]
There are arguments to be made in both directions.

On one hand, having your community all in one place means less confusion for anyone seeking out said community.

On the other hand, distributing the community far and wide makes it harder for any one event to destroy that community. (Have you ever heard that old saying about not keeping all of your eggs in the same basket?)

I'm...honestly not sure where I stand on the matter, but as long as this new Discord server continues to exist, I'll likely be around within it, so long as nothing bad happens.

**Comment by geometrikal** [comment]
The verify bot seems down

**Reply by nlogozzo** [reply]
The server is actually being discontinued
